modules/kubernetes-addons/agones/versions.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_version = ">= 1.0.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.72"
+    }
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = ">= 2.10"
+    }
+  }
+}
```

Comment:
Do we need versions file for each sub folder add-on? We have this defined in Kuberentes root module. The same version will implicitly applies to all the sub folders.

Reply:
Your 2nd comment is correct, but because we are treating them as modules, then they hit these rules https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/.tflint.hcl#L15-L21

I would propose that we move these out of modules within the addon directory and this would reduce all this boilerplate

Reply:
Ok.. so we need this to pass the tflint. I am fine with that.